### DISCUSSION TITLE
New into storage mining

### DISCUSSION THREAD

**Discussion by Maizerrr1** [filecoin-project/community#170] [Mining]
Hello. I'm new and I would like to get some Filecoins to start storage mining.

How many Filecoins are the minimum required to start being paid?

Where can I find a tutorial about how to step up the Filecoin storage mining device?

Thank you

**Comment by funktafide** [comment]
https://docs.filecoin.io 
This is the best place to get started.
https://fgas.io 
This will give you an idea based on the current gas fees how much FIL you will need to seal a specific amount of storage.

You need 10TiB of storage proving to be eligible for block rewards. It will currently take close to 100FIL to reach 10TiB.

**Comment by Maizerrr1** [comment]
Thank you for the reply.

 

So I need now 7.5 Filecoin per Terabyte this is 75 Filecoins minimum. Right? 

Does it change depending on Filecoin price evolution?

 

Can I withdraw this Filecoins at any moment?

 

Can I install it without being and informatics following instructions in “ <https://docs.filecoin.io> docs.filecoin.io”?

 

Another point is; why there’s such low people doing it when there are lots of people holding Filecoins?

 

Sorry for multiple questions.

Thank you again.

**Reply by rjan90** [reply]
Hey! :smile: 

`So I need now 7.5 Filecoin per Terabyte this is 75 Filecoins minimum. Right?`
Correct! But you should also have some FIL's for gas-fee's which can vary a lot, so having some extra FIL is probably a good idea.

`Does it change depending on Filecoin price evolution?`
No, but it changes depending on how much storage there is on the network. This article is quite good: https://ipfsforce-1751.medium.com/force-study-filecoins-second-growth-curve-collateral-decline-b784e974abd?source=social.tw

`Can I withdraw this Filecoins at any moment?`
No, your pledged collateral for a sealed CC-sectors is locked up for 540 days.

`Can I install it without being and informatics following instructions in “ <https://docs.filecoin.io> docs.filecoin.io”?`
To install Filecoin, you need to follow the steps in the docs.

`Another point is; why there’s such low people doing it when there are lots of people holding Filecoins?`
Being a storage provider on is an enteprising endeavor, and running a miner on Filecoin requires quite a lot of technical skills, as well as server-management, etc. Which is probably why there are a lot more people holding the token vs being a miner.

**Reply by Maizerrr1** [reply]
Great Rjan. Thank you.

Another question is related to the excel file calculator I found here:

It shows monthly costs for a initial investment of 200.000$.

It shows the following for the first 4months:
Storage Fee (PreCommit+ProveCommit) | $2,671.30 | $2,724.73 | $2,779.22 | $2,834.81
Pledged (Collateral) | $38,138.48 | $39,164.41 | $40,217.93 | $41,299.79
Total Storage Costs | $40,875.70 | $41,955.05 | $43,063.07 | $44,200.52

Why these costs without including Filecoin cost and hardware? Are these real?

An one more thing, do you have to keep buying new HDD? I thought once you get 10Tb it is all...

**Reply by funktafide** [reply]
Im not sure about that excel calculator.  I haven't seen it.  For you as a miner, you are really only concerned with the price of your hardware and the collateral you will need to pledge your sectors.  Hardware can be expensive depending on how big you plan to be.  A super small miner can be put together for $5,000.  A large one could cost you much much more.  The larger you are on the network, the more the rewards.  The initial roughly 100 or so you will need are just to get to 10TiB.  Getting to the 10TiB mark will allow you to win block rewards.  Before 10TiB you wont be eligible.  That's why you will need the 100 or so up front.  Just be aware that when you get to 10TiB, block rewards are not that often.  

You will want as much storage as you can support.  The more storage you have pledged to the network, the better the chances of mining a block reward.  The difference between how many rewards you will get at 10TiB vs how many you will get at 100TiB is pretty large.  

You can take a look at http://filfox.info and you can get an idea of how much storage big and small miners are pledging to the network.

**Reply by Okan1919** [reply]
Hi guys,
I have some questions, but I couldn' t find anywhere.
mining and storage independent, or not?
So can I work only storage?
How about network bandwith, dublex Gbps is enough for small (for 10-20 tib)? Only storing, or up-down all time?
Why ask this, because I have dublex Gbps connection :D ( my ISP give this only 11$/m )
I need only ram and hdd, other things I already have.
If I invest sealingh cost, can I get rewards?
